packages/date-picker/test/validation.test.js

```diff
@@ -288,30 +288,6 @@ describe('validation', () => {
         expect(validateSpy.called).to.be.false;
       });
     });
-
-    describe('autoOpenDisabled', () => {
```

Comment:
Since the logic no longer depends on autoOpenDisabled, I considered that these tests can be removed. Note, similar tests for the default `autoOpenDisabled = false` case still exist:

https://github.com/vaadin/web-components/blob/bfcbab4dcd22f28892dea2c9196ae81733964dde/packages/date-picker/test/validation.test.js#L102-L118